packages/FormElement/src/components/Label/Label.js

```diff
@@ -1,74 +1,68 @@
 import React from "react";
 import PropTypes from "prop-types";
-
 import useI18n from "@paprika/l10n/lib/useI18n";
-
-import labelStyles, { ruleStyles } from "./Label.styles";
 import Help from "../Help/Help";
 import { FormElementContext } from "../../FormElement";
-
-const propTypes = {
-  children: PropTypes.node.isRequired,
-  /** If "optional" text should be displayed beside the label  */
-  hasOptionalLabel: PropTypes.bool,
-  /** If "require" text should be displayed beside the label */
-  hasRequiredLabel: PropTypes.bool,
-  /** Help indicator */
-  help: PropTypes.node,
-  /** id for the element */
-  id: PropTypes.string,
-  /** Should label be hidden */
-  isVisuallyHidden: PropTypes.bool,
-
-  onClick: PropTypes.func,
-};
-
-const defaultProps = {
-  hasOptionalLabel: false,
-  hasRequiredLabel: false,
-  help: null,
-  id: null,
-  isVisuallyHidden: false,
-  onClick: () => {},
-};
+import * as sc from "./Label.styles";
 
 const Label = props => {
-  const { hasFieldSet, refLabel, labelId } = React.useContext(FormElementContext);
-  const { hasOptionalLabel, hasRequiredLabel, help, id, children, ...moreProps } = props;
-
+  const { help, helpA11yText, children, ...moreProps } = props;
+  const { hasFieldSet, isOptional, isRequired, isDisabled, labelId, refLabel } = React.useContext(FormElementContext);
   const I18n = useI18n();
 
   const labelProps = hasFieldSet ? { as: "legend" } : { as: "label" };
-
-  const a11yProps = { htmlFor: labelId, ref: refLabel };
+  const a11yProps = hasFieldSet ? { ref: refLabel } : { htmlFor: labelId, ref: refLabel };
 
   const renderQuestionRequirement = () => {
-    if (hasRequiredLabel) {
-      return I18n.t("formElement.required");
-    }
-    if (hasOptionalLabel) {
-      return I18n.t("formElement.optional");
+    if (isRequired) {
+      return `${I18n.t("formElement.required")}`;
     }
-  };
-
-  const renderHelp = () => {
-    if (help) {
-      return <Help>{help}</Help>;
+    if (isOptional) {
+      return `${I18n.t("formElement.optional")}`;
     }
-    return null;
   };
 
   return (
-    <div data-pka-anchor="form-element.label" css={labelStyles} {...labelProps} {...moreProps} {...a11yProps}>
+    <sc.Label data-pka-anchor="form-element.label" {...labelProps} {...moreProps} {...a11yProps}>
       {children}
-      {hasOptionalLabel || hasRequiredLabel ? <span css={ruleStyles}> {renderQuestionRequirement()}</span> : null}
-      {renderHelp()}
-    </div>
+      {isOptional || isRequired ? (
+        <>
+          &nbsp;<sc.Requirement>{renderQuestionRequirement()}</sc.Requirement>
+        </>
+      ) : null}
+      {help ? (
+        <>
+          &nbsp;
```

Comment:
Oh, I was trying my best to avoid an indent when the line breaks here.  It mostly works with `&nbsp;`.  The line breaks _after_ `the spacing and the part that breaks to a new line renders flush with the above line.